spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/cloud-foundry.adoc

```diff
@@ -30,25 +31,29 @@ If you want to fully disable the `/cloudfoundryapplication` endpoints, you can a
 By default, the security verification for `/cloudfoundryapplication` endpoints makes SSL calls to various Cloud Foundry services.
 If your Cloud Foundry UAA or Cloud Controller services use self-signed certificates, you need to set the following property:
 
+====
 [source,yaml,indent=0,subs="verbatim",configprops,configblocks]
 ----
 	management:
 	  cloudfoundry:
 	    skip-ssl-validation: true
 ----
+====
 
 
 
 [[actuator.cloud-foundry.custom-context-path]]
 === Custom Context Path
-If the server's context-path has been configured to anything other than `/`, the Cloud Foundry endpoints will not be available at the root of the application.
-For example, if `server.servlet.context-path=/app`, Cloud Foundry endpoints will be available at `/app/cloudfoundryapplication/*`.
+If the server's context-path has been configured to anything other than `/`, the Cloud Foundry endpoints are not available at the root of the application.
+For example, if `server.servlet.context-path=/app`, Cloud Foundry endpoints are available at `/app/cloudfoundryapplication/*`.
 
-If you expect the Cloud Foundry endpoints to always be available at `/cloudfoundryapplication/*`, regardless of the server's context-path, you will need to explicitly configure that in your application.
-The configuration will differ depending on the web server in use.
-For Tomcat, the following configuration can be added:
+If you expect the Cloud Foundry endpoints to always be available at `/cloudfoundryapplication/*`, regardless of the server's context-path, you need to explicitly configure that in your application.
+The configuration differs, depending on the web server in use.
+For Tomcat, you can add the following configuration:
 
+====
```

Comment:
What are the extra `====` lines for? I don't think we usually include them in source markup

Reply:
They add consistency. Some of our code listings have callouts to explain bits of code within the listing. When those are present, we want a box around both the listing and the callouts. We get that with `====` before the code listing and after the callouts. The trouble is that those code listings then look different than the other code listings, so I add the `====` around all the code listings, to make them all look the same.

Reply:
We've generally been avoiding callouts because they're a bit hard to manage with the extracted code that we use for all samples. The extra `====` add quite a lot of unnecessary markup, I think I'd rather we didn't add it to the Boot docs.

Reply:
With beginning and ending tags, you can add callouts to the code samples that you include from source files. The individual callouts also need to be in a comment at the end of the line. Callouts can be a big help to readers, as the individual steps of an algorithm or other details of a block of code can be pinpointed. I'd like to see more use of callouts for that reason.

Getting your code from the actual repositories is the gold standard, and I appreciate that you do it. We're getting better across the portfolio about not using "pseudo code," but there's a ways to go yet.

Reply:
I realize we can use them if we need to, but I still think adding extra `====` blocks around source that doesn't use them add noise that we'd rather not have. I think we should only add the extra `====` blocks if we have a block that actually uses callouts. Those will need to be different subs as well since by default we use `subs="verbatim"` and for those blocks we'll need `subs="verbatim,callouts"`

Reply:
The trouble is that then the code blocks with callouts look different than the code blocks without callouts. It just occurred to me that the better fix for all this is to extend the box for the code to include the callouts, so there's only a single box. I bet that's tricky to do, though.

Reply:
I've opened https://github.com/spring-projects/spring-boot/issues/27806 do deal with the callout issue. It's going to be a general problem and if we break it out into its own issue we can solve it later and continue with the general edits. We don't have any callouts at the moment anyway so we don't need to solve it urgently.